src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TIME_DISC = 50;
+  const MIDL_TIME_DISC = 20;
+  const PRICE = 40;
+  const LONG_TIME = 7;
+  const MIDL_TIME = 3;
```

Comment:
The constants are well-named and descriptive, making the code easier to understand. However, the names `LONG_TIME_DISC` and `MIDL_TIME_DISC` could be improved for consistency and readability to `LONG_TERM_DISCOUNT` and `MID_TERM_DISCOUNT` respectively.